types/backbone/index.d.ts

```diff
@@ -94,7 +95,8 @@ declare namespace Backbone {
         [selector: string]: string | {(eventObject: JQueryEventObject): void};
     }
 
-    class Events {
+    const Events: EventsClass;
+    abstract class EventsClass {
```

Comment:
Do you have a reason for not using an interface here?

Reply:
Was trying to avoid having to add signatures for Backbone.Events methods to everything, but I don't think there's a reasonable way around that.  Using an abstract class implies that somebody could extend it, which wouldn't work since Backbone.Events is an object.  I didn't consider that at the time, so thank you for pointing out that this likely wanted to be an interface, you were right.

Anyway, I don't think there's any way around adding Backbone.Events signatures to everything.  At least not one that I know of.  If there's a better way to handling mixins, at the type declaration level, that won't mislead the end user, I'm all ears.